package.json

```diff
@@ -29,21 +29,28 @@
     "fs-extra": "^4.0.1",
     "glob": "^7.1.2",
     "gulp": "github:gulpjs/gulp#4.0",
+    "gulp-rename": "^1.2.2",
+    "gulp-sourcemaps": "^2.6.0",
     "gulp-spawn-mocha": "^3.3.1",
     "lerna": "^2.0.0",
     "mocha": "^3.5.0",
     "path-extra": "^4.2.0",
     "pr-bot": "0.0.4",
     "rollup": "^0.45.2",
+    "rollup-plugin-babili": "^3.1.0",
     "rollup-plugin-commonjs": "^8.0.2",
     "rollup-plugin-multi-entry": "^2.0.1",
     "rollup-plugin-node-resolve": "^3.0.0",
     "rollup-plugin-replace": "^1.1.1",
+    "rollup-plugin-uglify": "^2.0.1",
     "rollup-stream": "^1.23.1",
     "sinon": "^2.4.1",
+    "uglify-es": "^3.0.27",
```

Comment:
> Minifier has been switched to uglify-es, this is so that we can minify private variables, which for the small amount of code got use from ~900B down to ~750B

Speaking to developers a little closer to Babili, although both it and uglify-es are maintained, the latter is considered a little more stable. I'm in favor of the switch in minifier here.